src/service/resource.js

```diff
@@ -265,7 +265,7 @@ export class Resource {
       this.state_ = ResourceState.NOT_LAID_OUT;
     }
     if (opt_callback) {
```

Comment:
From what I find yes. But @dvoytenko @sriramkrish85 you may want to confirm this